GameServer/Logic/Enet/ClientInfo.cs

```diff
@@ -11,114 +11,101 @@ namespace LeagueSandbox.GameServer.Logic.Enet
 {
     public class ClientInfo
     {
-        public bool keyChecked = false;
-        public bool versionMatch = true;
-        public long userId;
-        public int ticks = 0;
-        public int skinNo = 0;
-        public SummonerSpellIds[] summonerSkills = new SummonerSpellIds[2];
-        public string name, rank;
-        public short ribbon;
-        public int icon;
-        public TeamId team;
-
-        Champion champion;
-        Peer peer;
+        public bool KeyChecked = false;
+        public bool VersionMatch = true;
+        public long UserId;
+        public int SkinNo = 0;
+        public SummonerSpellIds[] SummonerSkills = new SummonerSpellIds[2];
+        public string Name, Rank;
+        public short Ribbon;
+        public int Icon;
+        public TeamId Team;
```

Comment:
These should be properties, so
`public TeamId Team { get; protected set; }` or something similar.